lib/philomena_web/controllers/api/json/posts_controller.ex

```diff
@@ -0,0 +1,72 @@
+defmodule PhilomenaWeb.Api.Json.PostsController do
+  use PhilomenaWeb, :controller
+
+  alias PhilomenaWeb.PostJson
+  alias Philomena.Topics.Topic
+  alias Philomena.Posts.Post
+  alias Philomena.Forums.Forum
+  alias Philomena.Repo
+  import Ecto.Query
+
+  def index(conn, %{"forums_id" => forum_id, "topics_id" => topic_id}) do
```

Comment:
Wrong id keys for singular name, should be `forum_id` and `topic_id`

Reply:
Now uses `short_name` and `slug` as per website proper